MeetupPlatformApi/Features/Meetups/GetMeetups/Filter/ConfigurationQuery/MeetupsFilterHelper.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.GetMeetups.Filter.ConfigurationQuery;
+
+using MeetupPlatformApi.Domain;
+using MeetupPlatformApi.Features.Meetups.GetMeetups.Filter.FilterSettings;
+using System.Linq;
+
+public static class MeetupsFilterHelper
+{
+    public static IQueryable<Meetup> GetMeetupsFilteredByFilterSettings(
+        IQueryable<Meetup> meetupsQuery,
+        MeetupsFilterSettings meetupFilterSettings)
+    {
+        meetupsQuery = meetupsQuery.Filter(meetupFilterSettings);
+        meetupsQuery = SortSelection(meetupsQuery, meetupFilterSettings.SortOption);
+        var skippedMeetups = (meetupFilterSettings.PageNumber - 1) * meetupFilterSettings.PageSize;
+
```

Comment:
I think it's redundant newline. Our actions have common goal.
But if you think `skippedMeetups` has other goal I recommend you to add newline between `meetupsQuery = SortSelection(meetupsQuery, meetupFilterSettings.SortOption);` and `var skippedMeetups = (meetupFilterSettings.PageNumber - 1) * meetupFilterSettings.PageSize;`